### PR TITLE
Update CONTRIBUTING.md to reflect new staging environment

### LINKED ISSUE #41: Create staging environment
Create a new CF app `dolores-staging` to deploy to `dolores-staging.18f.gov`. @mbland do you need anything from me in order to host `dolores-staging.18f.gov`
* [x] add `dolores-staging` CF app
* [x] add `dolores-staging` CF route 
* [x] hosting set up
* [x] create new MyUSA app `dolores-staging`
* [x] update CF env variables, including app keys from MyUSA app
* [x] update docs

### PR DESCRIPTION
**Pull request by gemfarmer**
Fixes #41

### CODE REVIEW

**Comment by gemfarmer** [comment]
@jessieay @ccostino could you look at this when you get a chance. Is it clear how to push to `dolores-app` vs `dolores-staging`?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -9,11 +9,15 @@ and ask that all contributors do the same.
 
 ### Git Protocol
 
-To contribute to this project, people internal to 18F can create a branch and submit a pull request. If you are external to 18F, you can fork the repository and submit a pull request that way.
+To contribute to this project, people internal to 18F can create a branch and 
+submit a pull request. If you are external to 18F, you can fork the repository 
+and submit a pull request that way.
 
-We are minimizing commits on the `master` branch by squashing commits and force pushing them to `master`. 
+We are minimizing commits on the `master` branch by squashing commits and force
```
Comment (jessieay) [inline CONTRIBUTING.md]: this is inaccurate and I changed it in https://github.com/18F/dolores-landingham-bot/pull/56

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -9,11 +9,15 @@ and ask that all contributors do the same.
...
+To contribute to this project, people internal to 18F can create a branch and 
+submit a pull request. If you are external to 18F, you can fork the repository 
+and submit a pull request that way.
 
-We are minimizing commits on the `master` branch by squashing commits and force pushing them to `master`. 
+We are minimizing commits on the `master` branch by squashing commits and force 
+pushing them to `master`. 
 
-For a more detailed walk through on how to do this, you can read thoughtbot's [Git Protocol](https://github.com/thoughtbot/guides/tree/master/protocol/git#write-a-feature) document.
+For a more detailed walk through on how to do this, you can read thoughtbot's [Git Protocol](https://github.com/thoughtbot/guides/tree/master/protocol/git#write-a-feature)
```
Comment (jessieay) [inline CONTRIBUTING.md]: hey, wanna rebase against master? there are some changes in this file that I just merged in.

**Comment by gemfarmer** [comment]
@jessieay I rebased. If you don't see any conflicts I'll squash and ship.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -7,15 +7,31 @@ To that end, we follow the [18F Code of
...
 
-To contribute to this project, people internal to 18F can create a branch and submit a pull request. If you are external to 18F, you can fork the repository and submit a pull request that way.
+To contribute to this project, people internal to 18F can create a branch and
+submit a pull request. If you are external to 18F, you can fork the repository
+and submit a pull request that way.
 
-We are minimizing commits on the `master` branch by squashing commits and force pushing them to `master`. 
+We are minimizing commits on the `master` branch by rebasing and squashing
+commits on branches before merging them into `master`. This will require you to
+force push on your branch (but not to `master`...please don't force push
```
Comment (jessieay) [inline CONTRIBUTING.md]: this is still a change that is on master - can you make sure your local master is up to date and then rebase against that?

**Comment by gemfarmer** [comment]
Closed and replaced with #64

**Pull request closed without merging**